lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -167,6 +183,35 @@ WasmBinaryReader::ReadFromModule()
      return wnLIMIT;
 }
 
+SectionCode
+WasmBinaryReader::SectionHeader()
+{
+    UINT len;
+    UINT sectionSize;
+    UINT idSize;
+
+    sectionSize = LEB128(len);
+    m_moduleState.count += len;
+    idSize = LEB128(len);
+    m_moduleState.count += len;
+    const char *sectionName = (char*)(m_pc);
+    m_pc += idSize;
+    m_moduleState.count += idSize;
+
```

Comment:
Right. Better to rename those variables then (size: bytesize, len: entries count)